kratos/geometries/geometry.h

```diff
@@ -988,6 +988,20 @@ class Geometry
         return mpGeometryData->LocalSpaceDimension();
     }
 
+    ///@}
+    ///@name Mathematical Informations
+    ///@{
+
+    /// Return polynomial degree of the geometry in a certain direction
+    virtual SizeType PolynomialDegree(IndexType DirectionIndex = 0) const
```

Comment:
I would call it `LocalDirectionIndex` and would remove the default argument.

This would let us to add another method (if needed in the future) if we want to get the polynomial order of the geometry in general.